### PR TITLE
OurTeam: Integrated Team API with frontend.

### LINKED ISSUE #695: Integrate Team API with frontend
API endpoint for `our-team` view is to be integrated

### PR DESCRIPTION
**Pull request by RishabhJain2018**
fixes #695  . @aka-jain @spyshiv @deshraj Please review the PR.

### CODE REVIEW

File: frontend/src/views/web/our-team.html [hunk 1]
```diff
@@ -2,124 +2,70 @@
     <main-header></main-header>
     <div class="content">
         <div class="grad-container">
-            <h4 class="team-heading">Team Awesome!</h4>
+            <h4 class="team-heading">{{ourTeam.coreTeamType}}</h4>
```
Comment (spyshiv) [inline frontend/src/views/web/our-team.html]: @RishabhJain2018 : This heading should be aligned center.

File: frontend/src/views/web/our-team.html [hunk 2]
```diff
@@ -2,124 +2,70 @@
...
-                        <div class="team-player">
-                            <div><strong class="text-dark-black">Deshraj Yadav</strong></div>
-                            <div class="text-med-black">Because I love <i class="fa fa-beer"></i></div>
-                        </div>
-                    </div>
-                </div>
-                <div class="col s12 l3 ">
+            </div>
+            <!-- This card is for Contributing Team -->
+            <h4 class="team-heading">{{ourTeam.contributingTeamType}}</h4>
```
Comment (spyshiv) [inline frontend/src/views/web/our-team.html]: this too...

Comment (RishabhJain2018) [inline frontend/src/views/web/our-team.html]: @spyshiv okay sure.

**Comment by RishabhJain2018** [comment]
@spyshiv @deshraj @aka-jain Please review the PR.

File: frontend/src/js/controllers/ourteamCtrl.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                            }
+                            vm.coreTeamDetails = coreTeamList;
+                            vm.contributingTeamDetails = contributingTeamList;
+                        }
+                    } else {
+                        vm.noTeamDisplay = "Team will be updated very soon !";
+                    }
+                }
+            },
+            onError: function(response) {}
```
Comment (spyshiv) [inline frontend/src/js/controllers/ourteamCtrl.js]: Please remove lint warning.

**Comment by deshraj** [comment]
Thanks @RishabhJain2018 for building this feature. :)

File: frontend/src/js/controllers/ourteamCtrl.js (hunk 3 above)
Comment (RishabhJain2018) [inline frontend/src/js/controllers/ourteamCtrl.js]: @spyshiv okay sure.

**Comment by RishabhJain2018** [comment]
@deshraj Please merge #827 as it completes this PR.

**Pull request merged**